doc/releases/changelog-dev.md

```diff
@@ -93,45 +57,88 @@
+  can be also accessed individually without initiating the class:
+
+  - The number of logical qubits with `qubit_cost`
+  - The number of non-Clifford gates with `gate_cost`
 
   ```python
   n = 100000
   eta = 156
   omega = 169.69608
   error = 0.01
-  qml.resource.FirstQuantization.qubit_cost(n, eta, omega, error)
   ```
   
   ```pycon
+  >>> qml.resource.FirstQuantization.qubit_cost(n, eta, omega, error)
   4377
+  >>> qml.resource.FirstQuantization.gate_cost(n, eta, omega, error)
+  3676557345574 
+  ```
+
+<h4>Differentiable error mitigation ⚙️</h4>
+
+* Differentiable zero-noise-extrapolation error mitigation is now available.
+  [(#2757)](https://github.com/PennyLaneAI/pennylane/pull/2757)
+
+  In zero-noise-extrapolation (ZNE), the noise of a circuit is artificially increased by _folding_
+  the circuit. By executing an expectation value with increasing noise, we can extrapolate back to
+  zero noise by means of a polynomial fit. More details on this can be found in the details section
+  of the [documentation](https://pennylane.readthedocs.io/en/latest/code/api/pennylane.transforms.fold_global.html)
+  and a soon-to-appear demo [(PennyLaneAI/qml/529)](https://github.com/PennyLaneAI/qml/pull/529).
+
+  The new feature in this release is that ZNE that is now *differentiable*, meaning that you can now elevate any variational
+  quantum algorithm that is written in PennyLane to a *mitigated* algorithm with improved results on noisy hardware while maintaining differentiability throughout.
+
+  In order to do so, use the `qml.transforms.mitigate_with_zne` transform on your QNode and provide the PennyLane proprietary
+  `qml.transforms.fold_global` folding function and `qml.transforms.poly_extrapolate` extrapolation function. Here is an example for a noisy simulation device
+  where we mitigate a QNode and still are able to compute the gradient:
+
+  ```python
+  # Describe noise
+  noise_gate = qml.DepolarizingChannel
+  noise_strength = 0.1
+
+  # Load devices
+  dev_ideal = qml.device("default.mixed", wires=n_wires)
+  dev_noisy = qml.transforms.insert(noise_gate, noise_strength)(dev_ideal)
+
+  scale_factors = [1, 2, 3]
+  @mitigate_with_zne(
+    scale_factors,
+    qml.transforms.fold_global,
+    qml.transforms.poly_extrapolate,
+    extrapolate_kwargs={'order': 2}
+  )
+  @qml.qnode(dev_noisy)
+  def qnode_mitigated(theta):
+      qml.RY(theta, wires=0)
+      return qml.expval(qml.PauliX(0))
+
+  theta = np.array(0.5, requires_grad=True)
+  grad = qml.grad(qnode_mitigated)
+  >>> grad(theta)
+  0.5712737447327619
   ```
 
-  In addition to the number of non-Clifford gates and logical qubits, some other quantities such as
-  the 1-norm of the Hamiltonian and double factorization of the second-quantized Hamiltonian can be
-  obtained either by initiating the classes or by directly calling the functions.
+<h4>More native support for parameter broadcasting with `DefaultQubit` devices 📡</h4>
```

Comment:
Are the two new features within this header correctly divided? I.e., is the first new feature

"DefaultQubit devices support param broadcasting natively"

and the second

"Parameter-shift gradients now allow for parameter broadcasting"

?

Reply:
Yes it is correct, those are the two bug changes. Do you think it would be better to merge them?

Reply:
No all good! Just want to make sure I'm understanding correctly 😄